connectivity/drivers/ble/FEATURE_BLE/TARGET_STM32WB/HCIDriver.cpp

```diff
@@ -490,6 +492,20 @@ class TransportDriver : public CordioHCITransportDriver {
                 tr_info("WIRELESS COPROCESSOR FW VERSION ID = %d.%d.%d", p_wireless_info->VersionMajor, p_wireless_info->VersionMinor, p_wireless_info->VersionSub);
                 tr_info("WIRELESS COPROCESSOR FW STACK TYPE = %d (ROM size 0x%x)", p_wireless_info->StackType, MBED_ROM_SIZE);
 
+#if STM32WB15xx
+                switch (p_wireless_info->StackType) {
+                    case INFO_STACK_TYPE_BLE_FULL:
+                        error("Wrong BLE FW\n");
+                        break;
+                    case INFO_STACK_TYPE_BLE_HCI:
+                        if (MBED_ROM_SIZE > 0x32800)  {
+                            error("Wrong MBED_ROM_SIZE with HCI FW\n");
+                        }
+                        break;
+                    default:
+                        tr_error("StackType %u not expected\n", p_wireless_info->StackType);
```

Comment:
just checking - two above are regular mbed  error, the last  one (default) is just trace error ?

Reply:
You're right, I will align in some coming PR

thx

Reply:
OK